xknx/devices/switch.py

```diff
@@ -93,6 +176,21 @@ async def set_off(self) -> None:
         """Switch off switch."""
         await self.switch.off()
 
+    @property
+    def current_power(self) -> Optional[float]:
+        """Return current power usage in W."""
+        return self._current_power.value  # type: ignore
+
+    @property
+    def total_energy(self) -> Optional[float]:
+        """Return total energy usage in kWh."""
+        return self._total_energy.value  # type: ignore
+
+    @property
+    def standby(self) -> Optional[bool]:
+        """Indicate if device connected to the switch is currently in standby."""
+        return self._standby.value  # type: ignore
```

Comment:
What does `standby` mean in this context?

Reply:
I was a bit torn on adding this, on how KNX works, but I added it for completeness sake.
The standby property is available in Home assistant, but might refer to switches of devices that actually report this. 
My thought was to include it and make it availble in Home assistant for devices that are also connected to KNX. 
For example I have a KNX enabled ventilation system that is powered by a metered KNX actuator. I this situation the standby can be provided by the ventilation system. Another example can be a Hue light connected to a KNX actuator.
If the examples are to particular then I can remove it. I am not tied to it :)

Reply:
I think Hue is a great example. I was always thinking of e.g. TV / HiFi stuff that is connected trough sockets - which comes down to the same - the standby state is probably not propagated directly to KNX.
For these we could use `standby` from HA by just configuring an entity_id. This could, maybe through a template, use its state or a state_attribute (extra_attribute) for the standby state. Of course a KNX binary_sensor entity_id can be used for this as well, so we would have everything covered.